ONSdigital/eq-author-app
# EAR-1834-create-questionnaire-modal-redesign

### What is the context of this PR?

This PR is linked to this [Jira ticket](https://jira.ons.gov.uk/browse/EAR-1834)

Redesigned the create questionnaire modal and removed the select questionnaire type as well as the toggle options.


### How to review

Author home page
Create questionnaire:
- Check that the create questionnaire button brings up the redesigned modal
- Ensure you can tab into all buttons and text fields
- Check that there is a validation error if no title is entered
- Enter a title and create a questionnaire, a new questionnaire should be created with an introduction page and a populated metadata array

Duplicate questionnaire:
- Check that the duplicate questionnaire button brings up the redesigned modal
- Ensure you are able to duplicate the questionnaire

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
